HomeWork 4/HomeWork 4/Program.cs

```diff
@@ -0,0 +1,222 @@
+﻿namespace HomeWork_4
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter length of array what you want to create at 0 for 255: ");
+
+            Console.WriteLine();
+
+            if (!byte.TryParse(Console.ReadLine(), out byte lengthArray) || lengthArray < 0 || lengthArray > byte.MaxValue)
```

Comment:
Wy  lengthArray > byte.MaxValue?

Reply:
if i use int.MaxVAlue will error with memory owersize. That's way i this parametr